src/store/creator/reducer.ts

```diff
@@ -33,5 +34,8 @@ export default createReducer<CreatorState>(initialState, builder =>
         })
         .addCase(setCursorAction, (state, { payload }) => {
             state.cursor = payload;
+        })
+        .addCase(updateStagedAction, (state, { payload }) => {
+            state.staged = merge({ message: '' }, state.staged, payload);
```

Comment:
If we keep `message` inside `staged, should it be non-nullable with `{ message: '' }` as its default value?